.github/workflows/test.yml

```diff
@@ -109,3 +109,36 @@ jobs:
     - name: Run the emulation tests
       run: |
         pytest --run-emulation test/test_emulation.py
+
+  test-emscripten:
```

Comment:
It looks like currently, the platform is pyodide, but the wheel platform tag is emscripten, similar to how we have `linux`->`manylinux`/`musllinux` currently. That could work too, I just wanted to check that makes sense to everyone.  (If that's the case, let's use the 'pyodide' terminology primarily.)

Reply:
There's an ongoing discussion about whether to call the platform emscripten or pyodide. See the discussion here:
https://github.com/pyodide/pyodide/pull/3823

Whether it should be like emscripten/linux ==> pyodide/manylinux or the other way around pyodide/linux ==> emscripten/manylinux is under debate. Of course neither is really like manylinux at all, it would be more like linux vs blessedlinux where blessedlinux wheels we can upload to pypi. The pypa folks I talked to at pycon seemed open to having Pyodide as a blessedemscripten platform.

Reply:
> I was thinking that we would standardize a Pyodide ABI and then put Pyodide in the platform tags. It might even be good to go the other way: by default tag the wheel with `emscripten_version_abiflags_wasm32` and switch to `pyodide_version_wasm32` if it is tested to match. Then `emscripten` can be like `linux`: a bit under-determined as a platform tag. And then `pyodide` isn't really like `manylinux` it's more like a specific ABI that is blessed by the PEP.

FWIW, this makes sense to me. Which points towards "pyodide" as the platform name and the term we'd use 99% of the time, which sounds good.